app/views/protips/index.html.haml

```diff
@@ -1,5 +1,5 @@
 - title protips_title
-- description protips_description
+-# - description protips_description
```

Comment:
What's the purpose of these changes?

Reply:
@usiegj00 The changes submitted are for the task related to META tags:
https://cl.ly/d26be953e613:

Meta-description too short
Meta description mission or empty
H1 tag missing or empty
Multiple H1 tags
Title too short

cc @freeminder

Reply:
There was no description meta tag in the rendered html at all.
![Screenshot from 2019-06-05 13-27-31](https://user-images.githubusercontent.com/5982702/58949764-db432b80-8795-11e9-89d0-943456d7fed3.png)
![Screenshot from 2019-06-05 13-28-00](https://user-images.githubusercontent.com/5982702/58949765-db432b80-8795-11e9-9197-c365388739f7.png)

Reply:
But you left commented code in the view, but didn't say why you commented it. Imagine someone new comes to look at this, will they understand why there is a commented line right there? Why did you comment it? Either add the comment so someone new will understand what you were thinking or remove the comment completely. Does that make sense? Try and look at this from someone that didn't know what you were doing or why. Be a good friend to your other developer teammates. :-P